aspnetcore/security/authorization/secure-data/samples/final15/Authorization/ContactRoleAuthorizationHandler.cs

```diff
@@ -16,7 +16,7 @@ public class ContactRoleAuthorizationHandler
                 return Task.FromResult(0);
             }
 
-            if (context.User.IsInRole(requirement.Name))
+            if (context.User.IsInRole(Constants.OperationRolePrefix + requirement.Name))
```

Comment:
@blowdart  can you help Joe out with this question?  Can you suggest names and code changes? This is a highly request sample customers need.